runestone/assess/js/mchoice.js

```diff
@@ -294,6 +294,11 @@ MultipleChoice.prototype.restoreAnswers = function (data) {
             }
         }
     }
+    if (this.multipleanswers) {
```

Comment:
It's the feedback that isn't being restored -- if students work half a page, then refresh, they won't know if the already-worked problems were correct or not. So, I just invoke the provide_feedback method on the stored answers to show this.